features/netsocket/emac-drivers/TARGET_WHD/interface/WhdAccessPoint.cpp

```diff
@@ -0,0 +1,48 @@
+#include <cstdlib>
```

Comment:
please add license header to any new code file

Reply:
license header added.